configuration/messageproperties/messages_fr.properties

```diff
@@ -2540,15 +2545,15 @@ mirebalaisreports.ncddataexport.description = Résultats (uniquement) sur les MN
 mirebalaisreports.HIVLTFUTracking.name=HIV LTFU Tracking
 mirebalaisreports.HIVLTFUTracking.description=HIV LTFU Tracking Data (only)
 
-mirebalaisreports.deliverydataexport.name=Données de livraison
-mirebalaisreports.deliverydataexport.description=Données de livraison (seulement)
-
-mirebalaisreports.outstandingpaperrecords.name=Dossiers papier exceptionnels
-mirebalaisreports.OutstandingPaperRecords.description=Dossiers papier exceptionnels
+mirebalaisreports.outstandingpaperrecords.name = Outstanding Paper Records
+mirebalaisreports.OutstandingPaperRecords.description = Outstanding Paper Records
 
```

Comment:
Not sure what's going on here, but I assume someone committed directly to the fr file and didn't go through Transifex.